duckduckgo/BrowserServicesKit
# Remote Messaging

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL: https://app.asana.com/0/72649045549333/1201884213179453/f
iOS PR:  https://github.com/duckduckgo/iOS/pull/1263
macOS PR: https://github.com/more-duckduckgo-org/macos-browser/pull/709
What kind of version bump will this require?: Major

**Optional**:

Tech Design URL: https://app.asana.com/0/481882893211075/1202238116542877/f
CC:

**Description**:
This PR adds remote messaging functionality. See the task for more details



**Steps to test this PR**:
1. Testing steps are included in the client implementation PRs.
2. Run unit tests and ensure they are green



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
